webapps/api/backend/decorators.py

```diff
@@ -81,7 +82,11 @@ def new_function(*args, **kw):
             if not ApiToolbox.is_token_in_roles(request.token, roles):
                 raise HttpForbiddenException(error_description='This call requires roles: {0}'.format(', '.join(roles)),
                                              error='invalid_roles')
-            return f(*args, **kw)
+            duration = time.time() - start
```

Comment:
No, as the goal is to time how long the security checking takes, not how long security checking *and* executing the function takes.